common/ast/src/parser/query.rs

```diff
@@ -136,10 +165,13 @@ pub fn alias_name(i: Input) -> IResult<Identifier> {
 }
 
 pub fn table_alias(i: Input) -> IResult<TableAlias> {
-    map(alias_name, |name| TableAlias {
-        name,
-        columns: vec![],
-    })(i)
+    map(
+        rule! { #alias_name ~ ( "(" ~ ^#comma_separated_list1(ident) ~ ")")? },
+        |(name, opt_columns)| TableAlias {
+            name,
+            columns: opt_columns.map(|col| col.1).unwrap_or_default(),
```

Comment:
Sorry, I misunderstood the rule. (The middle is columns instead of a single column)

```suggestion
            columns: opt_columns.map(|(_, columns,_)| columns).unwrap_or_default(),
```